packages/client/src/Dashboard.tsx

```diff
@@ -103,7 +103,7 @@ const useStyles = makeStyles(theme => ({
   }
 }))
 
-export default function Dashboard({ accountId }: Props) {
+export default function Dashboard({ accountId, navigate }: Props) {
```

Comment:
I switched from using the imported `navigate` function to getting `navigate` from props. `Dashboard` is a child component of a `Router` component. `Router` injects the `navigate` prop into its children. The `navigate` function in props uses the mock history set up in `LocationProvider`, which is important for testing.